docs/src/architecture/accounts.md

```diff
@@ -5,40 +5,52 @@ Miden aims to support expressive smart contracts via a Turing-complete language.
 The diagram below illustrates basic components of an account. In Miden every account is a smart contract.
 
 <p align="center">
-    <img src="../diagrams/architecture/account/Account_Definition.png">
+    <img src="../diagrams/architecture/account/Account_Definition.png" style="width: 50%;">
 </p>
 
 In the above picture, you can see:
 
 * **Account ID &rarr;** a unique identifier of an account which does not change throughout its lifetime
-* **Storage &rarr;** a user-defined data which can be stored in an account
-* **Nonce &rarr;** a counter which must be incremented whenever account state changes
+* **Storage &rarr;** user-defined data which can be stored in an account
+* **Nonce &rarr;** a counter which must be incremented whenever the account state changes
 * **Vault &rarr;** a collection of assets stored in an account
-* **Code &rarr;** a collection of functions which define an external interface for an account
+* **Code &rarr;** a collection of functions which define the external interface for an account
 
 ### Account ID
-~63 bits (1 Felt) long identifier for the account. The first three significant bits specify its type and the [storage mode](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html#account-storage-modes). There are four types of accounts in Miden:
+~63 bits (1 Felt) long identifier for the account. The four most significant bits specify its type and the [storage mode](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html#account-storage-modes). 
+
+#### Account types
+There are four types of accounts in Miden:
 
 | | Regular updatable account | Regular immutable account | Faucet for fungible assets | Faucet for non-fungible assets |
 |---|---|---|---|---|
-| **Description** | Will be used by most users for a wallet. Code specification and changes possible. | Will be used by most smart contracts. Once deployed code cannot be changed | Users can issue fungible assets and customize them. | Users can issue non-fungible assets and customize them. |
+| **Description** | For most users, e.g. a wallet. Code specification and change possible. | For most smart contracts. Once deployed code is immutable. | Users can issue fungible assets and customize them. | Users can issue non-fungible assets and customize them. |
 | **Code updatability** | yes | no | no | no |
 | **Most significant bits** | `00` | `01` | `10` | `11` |
 
-#### Example account ID (big-endian)
+#### Account storage mode
+Account data - stored by the Miden Node - can be public, private, or encrypted. The third and fourth most significant bits of the account ID specifies whether the account data is public `00`, encrypted `01`, or private `11`.
```

Comment:
If this is public-facing (i.e. not for internal developers), then I don't think we need to explain that level of detail.

And we should say something like "In Miden, there are three storage modes for accounts: public, private or encypted", then perhaps have a bullet point or subsection for each